### PR TITLE
docs(ui5-tabcontainer): add samples and documentation

### PR DESCRIPTION
**Pull request by kskondov**
Added a sample containing tabs with Icons and tabs without icons.
Fixed sample where count should be inline

Fixes: #6765

### CODE REVIEW

**Review by gmkv — APPROVED** [review]
LGTM. Let's wait for a review from @BorisDafov on the docs.

File: packages/main/src/Tab.ts
Comment (BorisDafov) [inline packages/main/src/Tab.ts:97]: I suggest:
In the cases when in the same time there are tabs with icons and tabs without icons, if a tab has no icon the "additionalText" is larger.

Comment (BorisDafov) [reply packages/main/src/Tab.ts:97]: Or:
 In the cases when in the same time there are tabs with icons and tabs without icons, if a tab has no icon the "additionalText" is displayed larger.

**Pull request merged**
